microsoft-edge/web-platform/local-network-access.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Control a website's access to the local network
+description: Local Network Access in Microsoft Edge enables end users to grant or deny a website access to their local network.  # key words before col 158
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/18/2025
+---
+# Control a website's access to the local network
+
+As a user of Microsoft Edge, you can use the Local Network Access feature to grant or deny a website access to your local network; that is, a network that uses a private IP address space.
+
+An enterprise administrator can configure the Local Network Access policies to grant or deny a website access to an enterprise's devices.
+
+The Local Network Access feature delivers secure, privacy-respecting browsing experiences, while maintaining compatibility with modern web standards.
+
+This feature requires Microsoft Edge 140 or later.
+
+
+<!-- ====================================================================== -->
+## What is Local Network Access?
+
+_Local Network Access_ is a feature in Microsoft Edge that restricts a website's ability to send requests to servers on a user's local network.  This feature displays a prompt to users (website visitors), requiring the user to grant or deny the website permission before these requests can be made.
+
+The ability of a website to request this permission from users is restricted to secure contexts.
+
+
+<!-- ====================================================================== -->
+## How to Enable Local Network Access in Microsoft Edge
+
+Users (website visitors using Microsoft Edge) can enable and manage Local Network Access permissions through the Microsoft Edge browser settings.
+
+To grant or deny a website permission to access your local network:
+
+1. In Microsoft Edge, go to `edge://flags/#local-network-access-check`.
+
+1. In the dropdown list for **Local Network Access Checks**, select **Enabled (Blocking)**.
+
+   Or, select **Enabled**, which will make the website display a warning, rather than blocking the network request.
+
+1. Click the **Restart** button in the lower right.
+
+1. In Microsoft Edge, select **Settings and more** (![Settings and more icon](./local-network-access-images/settings-and-more-icon.png)) > **Settings**.
+
+   The **Settings** page opens.
+
+1. Select **Privacy, search, and services** > **Site permissions** > **All permissions**.
+
+   The **All permissions** page is displayed:
+
+   ![The All permissions page in Edge Settings](./local-network-access-images/settings-all-permissions.png) 
```

Comment:
redo png: scroll up to top of page on right so that the bold heading of page **All permissions** is visible.

not sure i could do this layout

@murphys0709 

* Internal preview: https://review.learn.microsoft.com/microsoft-edge/web-platform/local-network-access?branch=pr-en-us-3553#how-to-enable-local-network-access-in-microsoft-edge
* External preview: https://github.com/murphys0709/edge-developer/blob/patch-2/microsoft-edge/web-platform/local-network-access.md#how-to-enable-local-network-access-in-microsoft-edge